.github/workflows/ci.yml

```diff
@@ -28,7 +28,9 @@ jobs:
       - name: Run static checks
         env:
           COMMIT_COUNT: ${{ github.event.pull_request.commits  }}
-        run: python3 setup.py test --select=static-checks
+        run: |
+          export PATH=/github/home/.local/bin:$PATH
+          python3 setup.py test --select=static-checks
```

Comment:
Hello @richtja, as we can ensure pylint will be installed during `make requirements-dev`, can we modify those shell scripts to use tools directly like isort.sh/style.sh, without `which pylint`, so we don't need this change I think.

Reply:
Or use `pylint-3`, seems it will installed to `/usr/bin/`

Reply:
Hi @PaulYuuu thanks for your help. I removed the `pylint-3` because of [this comment](https://github.com/avocado-framework/avocado/pull/5959#issuecomment-2188983220). 

The reason for `export` is that those scripts fails in CI without any logs. I am not sure why it fails with `python3 -m pylint` therefore I'm moving this to draft, and I will add more verbosity to it to find out what is going on in CI environment.